components/mpas-ocean/cime_config/buildnml

```diff
@@ -560,6 +568,18 @@ def buildnml(case, caseroot, compname):
                 lines.append('</stream>')
                 lines.append('')
 
+            if ocn_tidal_mixing == 'true':
+                lines.append('<stream name="tidal_velocity_rms_parameterized"')
+                lines.append('       type="input"')
+                lines.append('       io_type="{}"'.format(ocn_pio_typename))
+                lines.append('       filename_template="{}/ocn/mpas-o/{}/{}"'.format(din_loc_root, ocn_mask, u_tidal_rms_file))
+                lines.append('       input_interval="initial_only">')
```

Comment:
@irenavankova, I think it was right before without the `>`.  Were you having trouble?

Reply:
I was but for a different reason - see slack and library problems. I just noticed that on the 

lines.append('<stream name="transectMasksInput"')

there was input_interval="initial_only">' but at other place in the code it is sometimes input_interval="initial_only" '